api/python/t4/snapshots.py

```diff
@@ -147,8 +182,8 @@ def download_bytes_from_snapshot(src, snapshothash):
     return download_bytes(src, version=obj_rec['VersionId'])
 
 class PhysicalKeyType(Enum):
```

Comment:
Do we actually need `PhysicalKeyType` at all if we're using URIs?

Reply:
I think we want an enum of URI schemes instead? maybe that is for a separate PR.

Reply:
Probably not